parkhangs-webapp/src/components/EventForm.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
-import { connect } from 'react-redux'
-import { addEvent } from 'features/parks/parksSlice'
+import {connect} from 'react-redux'
```

Comment:
can you please leave the spaces it looks better that way and is pretty common practice.

Reply:
yes, my IDE (Webstorm) removes the space